lib/job.rb

```diff
@@ -283,7 +277,7 @@ def url
   end
 
   def abort
-    redis.hset(ident, 'abort_requested', true)
+    redis.hset(ident, 'aborted', true)
```

Comment:
For now, the bot should probably set both `abort_requested` and `aborted`.  Older versions of the wpull hooks look for the former.